cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -550,6 +590,125 @@ impl pallet_bridge_relayers::Config for Runtime {
+	type Token = Balances;
+	type Reward = Reward;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type XcmSender = XcmRouter;
+	#[cfg(feature = "runtime-benchmarks")]
+	type XcmSender = DoNothingRouter;
+	type WeightInfo = weights::snowbridge_inbound_queue::WeightInfo<Runtime>;
+	type GatewayAddress = GatewayAddress;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Helper = Runtime;
+}
+
+parameter_types! {
+	pub const MaxMessagePayloadSize: u32 = 2048;
+	pub const MaxMessagesPerBlock: u32 = 32;
+}
+
+impl snowbridge_outbound_queue::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Hashing = Keccak256;
+	type MessageQueue = MessageQueue;
+	type MaxMessagePayloadSize = MaxMessagePayloadSize;
+	type MaxMessagesPerBlock = MaxMessagesPerBlock;
+	type WeightInfo = ();
+}
+
+#[cfg(not(feature = "beacon-spec-mainnet"))]
+parameter_types! {
+	pub const ChainForkVersions: ForkVersions = ForkVersions {
+		genesis: Fork {
+			version: [0, 0, 0, 1], // 0x00000001
+			epoch: 0,
+		},
+		altair: Fork {
+			version: [1, 0, 0, 1], // 0x01000001
+			epoch: 0,
+		},
+		bellatrix: Fork {
+			version: [2, 0, 0, 1], // 0x02000001
+			epoch: 0,
+		},
+		capella: Fork {
+			version: [3, 0, 0, 1], // 0x03000001
+			epoch: 0,
+		},
+	};
+	pub const MaxExecutionHeadersToKeep:u32 = 1000;
+}
+
+#[cfg(feature = "beacon-spec-mainnet")]
+parameter_types! {
+	pub const ChainForkVersions: ForkVersions = ForkVersions {
+		genesis: Fork {
+			version: [0, 0, 16, 32], // 0x00001020
+			epoch: 0,
+		},
+		altair: Fork {
+			version: [1, 0, 16, 32], // 0x01001020
+			epoch: 36660,
+		},
+		bellatrix: Fork {
+			version: [2, 0, 16, 32], // 0x02001020
+			epoch: 112260,
+		},
+		capella: Fork {
+			version: [3, 0, 16, 32], // 0x03001020
+			epoch: 162304,
+		},
+	};
+	pub const MaxExecutionHeadersToKeep:u32 = 8192 * 2;
+}
+
+impl snowbridge_ethereum_beacon_client::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type ForkVersions = ChainForkVersions;
+	type MaxExecutionHeadersToKeep = MaxExecutionHeadersToKeep;
+	type WeightInfo = weights::snowbridge_ethereum_beacon_client::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	// TODO: placeholder value - choose a real one
+	pub const MaxUpgradeDataSize: u32 = 1024;
+	pub const RelayNetwork: NetworkId = Rococo;
+}
+
+impl snowbridge_control::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type OwnParaId = ParachainInfo;
+	type OutboundQueue = EthereumOutboundQueue;
+	type MessageHasher = BlakeTwo256;
+	type WeightInfo = weights::snowbridge_control::WeightInfo<Runtime>;
+	type MaxUpgradeDataSize = MaxUpgradeDataSize;
+	type AgentHashedDescription = AgentHashedDescription;
+	type UniversalLocation = UniversalLocation;
+	type RelayLocation = RelayLocation;
+	type AgentOrigin = EnsureXcm<Everything>;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type ChannelOrigin = EnsureXcm<AllowSiblingsOnly>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type ChannelOrigin = EnsureXcm<Everything>;
```

Comment:
To get the benchmarks to pass. `try_successful_origin` doesn't work with the `AllowSiblingsOnly`, so for benchmarks allow any XCM origin.